src/app/campaign/availability/availability.component.ts

```diff
@@ -1,59 +1,92 @@
-import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Component, ChangeDetectionStrategy, Input, Output, EventEmitter } from '@angular/core';
 import { Flight, Availability, InventoryZone } from '../../core';
 
 @Component({
   selector: 'grove-availability',
   template: `
```

Comment:
This template is getting a bit long, but also this file is mostly template since the data is wrangled in the rolled up observable. 🤷‍♀